bears/python/PyUnusedCodeBear.py

```diff
@@ -21,7 +21,12 @@ def run(self, filename, file):
         - Unneeded pass statements.
         - Unneeded builtin imports. (Others might have side effects.)
         """
-        corrected = autoflake.fix_code(''.join(file)).splitlines(True)
+        code = ''.join(file)
+        corrected = autoflake.fix_code(code,
+                                       additional_imports=None,
+                                       remove_all_unused_imports=True,
+                                       remove_unused_variables=True
+                                       ).splitlines(True)
```

Comment:
you should also write some tests for this too, to prove that it works :) cuz the code looks fine ..